services/src/main/java/org/exoplatform/news/connector/NewsSearchConnector.java

```diff
@@ -66,6 +67,10 @@ public List<SearchResult> search(NewsFilter filter,
             repositoryService.getCurrentRepository());
 
     if (filter.getSearchText() != null) {
+      // search text composed by one keyword maybe it is the tag text
+      if (filter.getSearchText().split(" ").length <= 1){
```

Comment:
We need to be sure if the text is a tag or a simple text, we can not guess it based on the number of words in the query